README.md

```diff
@@ -10,7 +10,8 @@ How to build
 
 ```
 % git submodule update --init --recursive
-% curl -L https://cpanmin.us | perl - --installdeps --notest .
+% curl -sL https://cpanmin.us | perl - --self-upgrade # add --sudo if needed
```

Comment:
Maybe "add --sudo when installing to system perl include path"?

"if needed" is not that meaningful unless we describe what "--sudo" means.